### PR TITLE
Function framework standalone feature

### PR DESCRIPTION
**Pull request by pwittrock**

### CODE REVIEW

File: kyaml/fn/framework/framework.go [hunk 1]
```diff
@@ -173,18 +220,142 @@ func (r *ResourceList) Write() error {
...
+		}
+		nodes, err := (&kio.ByteReader{Reader: bytes.NewBufferString(s)}).Read()
+		if err != nil {
+			// create the debug string
+			lines := strings.Split(s, "\n")
+			for j := range lines {
+				lines[j] = fmt.Sprintf("%03d %s", j+1, lines[j])
+			}
+			s = strings.Join(lines, "\n")
+			return errors.WrapPrefixf(err, "failed to parse rendered template into a resource:\n%s\n", s)
```
Comment (kwyn) [inline kyaml/fn/framework/framework.go:289]: Excited about this feature addition.

**Review by kwyn — APPROVED** [review]
LGTM: one small comment.

File: kyaml/fn/framework/framework.go [hunk 2]
```diff
@@ -82,10 +91,31 @@ type ResourceList struct {
...
 
 // Read reads the ResourceList
 func (r *ResourceList) Read() error {
+	// parse the inputs from the args
+	if len(r.Args) > 0 && !r.DisableStandalone {
+		// write the files as input
+		var buf bytes.Buffer
+		for i := range r.Args {
+			if i == 0 {
+				continue
```
[the remark this answers is not in the document]
Comment (pwittrock) [reply kyaml/fn/framework/framework.go:108]: Done

**Comment by pwittrock** [comment]
Add lgtm from Kwyn

**Pull request merged**